packages/ui/src/components/Dialog/index.module.scss

```diff
@@ -0,0 +1,58 @@
+.utrecht-dialog {
+  --utrecht-dialog-min-block-size: 200px;
+  --utrecht-dialog-min-inline-size: 50%;
+  --utrecht-dialog-border-radius: 3px;
+  --utrecht-dialog-backdrop-background-color: rgb(0 0 0 / 30%);
+  --utrecht-dialog-header-border-block-end-color: #e4e7ec;
+  --utrecht-dialog-header-border-block-end-width: 1px;
+  --utrecht-dialog-header-padding-inline-start: 16px;
+  --utrecht-dialog-header-padding-inline-end: 16px;
+  --utrecht-dialog-header-padding-block-start: 20px;
+  --utrecht-dialog-header-padding-block-end: 20px;
+  --utrecht-dialog-body-padding-inline-start: 16px;
+  --utrecht-dialog-body-padding-inline-end: 16px;
+  --utrecht-dialog-body-padding-block-start: 24px;
+  --utrecht-dialog-body-padding-block-end: 24px;
+  --utrecht-dialog-footer-padding-inline-start: 16px;
+  --utrecht-dialog-footer-padding-inline-end: 16px;
+  --utrecht-dialog-footer-padding-block-start: 24px;
+  --utrecht-dialog-footer-padding-block-end: 24px;
```

Comment:
Hebben we niet een file in de strapi repo waar we deze CSS variables op `.utrecht-theme` kunnen zetten?

Reply:
In de PDC-frontend hebben we de CSS-variabelen ingesteld in de [globals.css](https://github.com/frameless/strapi/blob/main/apps/pdc-frontend/src/styles/globals.css#L9 ). Ik denk dat we voor nu deze waarden als standaardwaarden kunnen gebruiken.